src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const char of actions) {
+    if (char.type === 'addProperties') {
```

Comment:
use switch case instead if/if/if

Reply:
also you can destruct your object
const {type, extraData, keysToRemove, type} = char